custom-commands/hospitality/deployment/DEPLOYREADME.md

```diff
@@ -1,32 +1,76 @@
+* Powershell 6.0 or greater. [Download Powershell here](https://github.com/PowerShell/PowerShell/releases). 
+    * On Windows, download and run an .msi file (e.g. "PowerShell-7.0.0-win-x64.msi")
+* Azure CLI. [Insall Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.4.0 or higher.
+* .NET Core SDK 2.1 or higher. [Install SDK here](https://docs.microsoft.com/en-us/dotnet/core/install/sdk?pivots=os-windows).
+
+## What are you deploying?
+ 
+ TODO: Need diagram of CC and Visualization, and a description of the components and how they interact
 
 ## Deploying Azure Resources
-Clone the repository and change directory so you are in the root/custom-commands/hospitality/deployment folder.
+Clone the repository and change directory so you are in the \<<repo-root\>>\custom-commands\hospitality\deployment folder.
 
-You will need to unrestrict powershell's script execution policy by running the following in an administrator powershell
+You will need to unrestrict powershell's script execution policy by running the following in an administrator powershell:
+```cmd
     Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
+```
 This will reset the policy once this powershell session has ended.
 
-Call az login to log your powershell into Azure. Run the powershell script. Replace "mynewresource" with the name you would like. Try to make this name unique as it is required to be GLOBALLY unique. It will be used to generate a url. Note: This will use your default Azure subscription if you have more than one.
-
+Call az login to log your powershell into Azure. If you have more than one Azure login credentials, you will be prompted to selected one or them:
+```cmd
     az login
-    ./deployAll.ps1 -resourceName mynewresource -region westus2
-
-wait...
-
-You should be good to go! The script will print out some resources you will need to start using your hospitality deployment.
+```
+ Run the following powershell script. 
+ * Replace "MyResourceGroupName" with the an Azure Resource Group name of your choice. This name should be no more than 19 characters, alphanumeric only. Make sure an Azure resource group by this name does not already exist. This name will also be used to construct names of all the Azure resources and URL that will be associated with the visualization of the Custom Commands in your Azure subscription. Some of these Azure resource names (and URL) need to be globally unique, so the script will append a random number to the name you selected
+ * Replace "westus" with an Azure region near you, form the list of [Voice Assistant supported regions](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#voice-assistants). Read more about Azure Regions [here](https://azure.microsoft.com/en-us/global-infrastructure/regions/). Note that if you are using a free trial Azure subscirption, you are limited to Azure regions westus or northeurope
```

Comment:
typo in subscirption.

Also it is tricky/annoying about the region lock for the App service. It isn't that it is a free trial, it is because we are creating a free SKU which is not available everywhere for some reason. We could change it to a paid SKU since the trial has an amount of "credit" but I was trying to keep it as low cost as possible.

Reply:
typo fixed, thanks.